aspnetcore/security/authentication/azure-ad-b2c-webapi.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Cloud authentication in web APIs with Azure Active Directory B2C
+author: camsoper
+description: Discover how to set up Azure Active Directory B2C authentication with ASP.NET Core Web API. Test the authenticated web API with Postman.
+ms.author: casoper
+manager: wpickett
+ms.date: 01/25/2018
+ms.topic: tutorial
+ms.technology: aspnet
+ms.prod: asp.net-core
+uid: security/authentication/azure-ad-b2c-webapi
+custom: mvc
+---
+# Cloud authentication in web APIs with Azure Active Directory B2C
+
+By [Cam Soper](https://twitter.com/camsoper)
+
+[Azure Active Directory B2C](/azure/active-directory-b2c/active-directory-b2c-overview) (Azure AD B2C) is a cloud identity management solution for your web and mobile apps. The service provides authentication for apps hosted in the cloud and on-premises. Authentication types include include individual accounts, social network accounts, and federated enterprise accounts.  Additionally, Azure AD B2C can provide multi-factor authentication with minimal configuration.
+
+Since web APIs have no user interface, they are unable to redirect the user to a secure token service like Azure AD B2C. Instead, the API is passed a bearer token from the calling app, which has already authenticated the user with Azure AD B2C. The API then validates the token without direct user interaction.
+
+In this tutorial, learn how to:
+
+> [!div class="checklist"]
+> * Create an Azure Active Directory B2C tenant
+> * Register a Web API in Azure AD B2C
+> * Use Visual Studio to create a Web API configured to use the Azure AD B2C tenant for authentication
+> * Configure policies controlling the behavior of the Azure AD B2C tenant
+> * Use Postman to simulate a web app which will present a login dialog, retrieve a token, and then use it to make a request against the web API
```

Comment:
We try to avoid the word "will" in aspnet/Docs:
~~Use Postman to simulate a web app which will present a login dialog, retrieve a token, and then use it to make a request against the web API~~
Use Postman to simulate a web app which presents a login dialog, retrieves a token, and uses it to make a request against the web API

Reply:
God, I wrestled with that sentence SO MUCH.  Yours is MUCH better.

Reply:
Scott's change is perfectly fine but you can even say "Use Postman to test authentication with the API" - it's a bit less scary and easier sounding. OTOH if you are doing this do you expect it to be not scary and easy? Either way. Your call.